services.json

```diff
@@ -746,7 +746,7 @@
     "models": "s3/2006-03-01",
     "dependencies": {
       "aws-sdk-kms": "~> 1",
-      "aws-sigv4": "~> 1.1"
+      "aws-sigv4": "~> 1.4"
```

Comment:
Discussed offline, I think we need to bump this generally for all services.

Reply:
Discussed offline - we will leave these as 1.1 for all non s3 gems.